upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -476,8 +476,8 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		key := "networking.projectcalico.org"
 		versions := map[string]string{
 			"pre-k8s-1.6": "2.4.1",
-			"k8s-1.6":     "2.6.6",
-			"k8s-1.7":     "2.6.6",
+			"k8s-1.6":     "2.6.8",
```

Comment:
Thanks for the PR @tmjd. 

Should these versions not be 2.6.7?

Reply:
Probably, I had duplicated what was done with the `pre-k8s-1.6` as it has +1 on the minor.  I think maybe that one was bumped to 2.4.1 when it should have been 2.4.0.

Reply:
Would it be safe to change the `pre-k8s-1.6` to 2.4.0?  I'm not sure if that would break anything.

Reply:
It looks like the minor +1 convention was specific to (https://github.com/kubernetes/kops/commit/148a65e1ad9041ceb1d46932fc6214d1a61a58e6). So we should definitely leave the pre-k8s-1.6 version as-is.

If I'm understanding the original intention correctly, the new versions should be 2.6.7-kops.1 for both canal and calico. (As a preemptive step for the latter). Then if future kops-specific patches are necessary, we'll just bump the suffix. 

@justinsb can you please confirm this is the versioning convention you had in mind in https://github.com/kubernetes/kops/commit/148a65e1ad9041ceb1d46932fc6214d1a61a58e6?